operations_on_datastructures/Array Left Rotation.cpp

```diff
@@ -1,39 +1,31 @@
 #include <iostream>
-using namespace std;
-int main()
-{
-	int n, k;
-	cout << "Enter size of array=\t";
-	cin >> n;
-	cout << "Enter Number of indeces u want to rotate the array to left=\t";
-	cin >> k;
-	int a[n];
-	cout << "Enter  elements of array=\t";
-	for (int i = 0; i < n; i++)
-	{
-		cin >> a[i];
-	}
-	int temp = 0;
-	for (int i = 0; i < k; i++)
-	{
-		temp = a[0];
-		for (int j = 0; j < n; j++)
-		{
-			if (j == n - 1)
-			{
-				a[n - 1] = temp;
-			}
-			else
-			{
-				a[j] = a[j + 1];
-			}
-		}
-	}
-	cout << "Your rotated array is=\t";
-	for (int j = 0; j < n; j++)
-	{
-		cout << a[j] << " ";
-	}
-	getchar();
-	return 0;
+int main() {
```

Comment:
Please provide basic description for your program. You have removed all the comments.

Reply:
The file Array Left Rotation.cpp is not made by me, it already didn't had any comments. I just made some changes to it otherwise it was same as before in the repos. I did however make array_left_rotation_efficient.cpp  is rotates the array d times to the left. Where d is user input. 
The Error that is coming is this,
`array_left_rotation_efficient.cpp:(.text+0x0): multiple definition of `main'`
I don't know how to solve it as there is no multiple definition of main in my code. 
Want your help in that.

Reply:
Have you checked the code on local machine. Is it executing there?

Reply:
Yup! Executing fine on my machine.
![image](https://user-images.githubusercontent.com/42692727/84120895-11c02000-aa54-11ea-99d6-058c491cd708.png)